corio.py

```diff
@@ -422,8 +422,9 @@ def main(options):
                                                 terminated_tests=test_ids)
                 schedule.cancel_job(sched_job)
                 if options.support_bundle:
-                    collect_upload_sb_to_nfs_server(MOUNT_DIR, sb_identifier,
-                                                    max_sb=CORIO_CFG['max_no_of_sb'])
+                    resp = collect_upload_sb_to_nfs_server(MOUNT_DIR, sb_identifier,
+                                                           max_sb=CORIO_CFG['max_no_of_sb'])
+                    LOGGER.info("collect support bundles response: %s", resp)
```

Comment:
Is this log statement really needed? `collect_upload_sb_to_nfs_server` already logs SB list on success and error on failure.

Reply:
Yes, this will appear in root directory.